source/nordic_sdk/components/ble/common/ble_srv_common.h

```diff
@@ -68,8 +69,6 @@
 #define BLE_UUID_RUNNING_SPEED_AND_CADENCE                       0x1814     /**< Running Speed and Cadence service UUID. */
 #define BLE_UUID_SCAN_PARAMETERS_SERVICE                         0x1813     /**< Scan Parameters service UUID. */
 #define BLE_UUID_TX_POWER_SERVICE                                0x1804     /**< TX Power service UUID. */
-#define BLE_UUID_IPSP_SERVICE                                    0x1820     /**< Internet Protocol Support service UUID. */

```

Comment:
Interesting this UUID has been removed from the Nordic SDK. I wonder if that has something to do with their new IP strategy.